webapp/static/interop-data.json

```diff
@@ -446,44 +445,44 @@
             },
             "interop-2023-mathfunctions": {
                 "description": "Math Functions",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-mathfunctions", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Functions#math_functions",
+                "spec": "https://w3c.github.io/csswg-drafts/css-values-4/#math",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-mathfunctions",
```

Comment:
all tests are in `css/css-values`

Reply:
With apologies, what should the URI be exactly?

Reply:
Oh I meant the `tests` page: `"tests": "/results/css/css-values?[...]`, same for other places i made a similar comment

Reply:
oic